apps/trading/components/constants.ts

```diff
@@ -2,6 +2,7 @@ import { t } from '@vegaprotocol/i18n';
 
 export const THROTTLE_UPDATE_TIME = 500;
 export const ONBOARDING_VIEWED_KEY = 'vega_onboarding_viewed';
+export const STARTED_STEPS_KEY = 'vega_started_steps_viewed';
```

Comment:
chore (blocking): This doesn't appear to be used anywhere

Reply:
removed